dev/eslint-plugin-guardian-frontend/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-plugin-guardian-frontend",
   "main": "index.js",
-  "version": "2.0.0",
+  "version": "3.1.0",
   "scripts": {
     "test": "jest"
```

Comment:
Unrelated, do you think we should add Jest as a `devDependency` here? This script will only work if Jest is installed globally